### ISSUE TITLE
WMS failure if format is set under the WMS options 

### ISSUE THREAD

**Issue by rgtaylor1989** [mapbox/leaflet-image#70]
```
  if (zoom > layer.options.maxZoom ||
        zoom < layer.options.minZoom ||
        // mapbox.tileLayer
      (layer.options.format && !layer.options.tiles)) {
        return callback();
    }
```

This is from leaflet-image.js around line number 80.

In the code above, what is the point of (layers.options.format && !layer.options.tiles)?
This part of the code blocks any wms calls a developer makes if they have a wms call that contains the format option 

EX:

```
const Parcels = L.tileLayer.wms('http://services.nconemap.gov/arcgis/services/NC1Map_Parcels/MapServer/WMSServer', {
            layers: 'Parcels',
            format: 'image/png',
            transparent: true
        });
```

From this example, layers.options.format is 'image/png' and layer.options.tiles is 'undefined' causing the callback to pop and preventing the wms from being populated.

leaving the orginal code
![7ktdnoxjq2vaaaaaelftksuqmcc](https://cloud.githubusercontent.com/assets/7094059/14894072/e24f7bdc-0d3f-11e6-9e31-cc34a764f8d3.png)

removing this portion of code
![download 14](https://cloud.githubusercontent.com/assets/7094059/14894039/b4a554ae-0d3f-11e6-96b4-de8745b676c9.png)

**Comment by bloigge** [comment]
Still have a problem using a wms tileLayer in nodejs. When I load the tiles using the format option I don't get any tiles. Without using the format option I get jpeg tiles (but I need png).

```
var tileLayer = new L.tileLayer.wms('http:/mygeoserver.com:8080/geoserver/wms?', { 
        format: 'image/png',
        layers: 'DB:Layer'
}).addTo(map);
```

**Comment by rgtaylor1989** [comment]
**Still have a problem using a wms tileLayer in nodejs. When I load the tiles using the format option I don't get any tiles.**

I haven't tried to use this plugin since the update was implemented. I found this example using html2canvas because I needed some divs that were not supported by this plugin

http://jsfiddle.net/oo2yms1h/5/

You might be able to use phantomjs also 

http://phantomjs.org/screen-capture.html

**Without using the format option I get jpeg tiles (but I need png).**

http://leafletjs.com/reference-1.0.0.html#tilelayer-wms-format

This is the default in leaflet. See link above.
